releases.json

```diff
@@ -141,6 +141,14 @@
       "1.0.0-1"
     ]
   },
+  "aklomp-base64": {
+    "dependency_names": [
+      "aklomp-base64"
```

Comment:
The dependency name, on the other hand, should match upstream's pkg-config name.  Upstream doesn't have one, but the CMake config is `base64-config.cmake`.  So this should probably be `base64`.